### ISSUE TITLE
Aligning on event median

### ISSUE THREAD

**Issue by ClaraBesserer** [ahwillia/affinewarp#19]
Hello,
I'm trying to apply warping to my data, following the 'rat motor cortex' tutorial. 
I noticed that it always align to the max isi (for tap2 in the exemple). It is not ideal if my event distribution is polluted by some abnormaly long isi. Is it possible to align on the isi mean or median?

![Figure 2021-12-15 095931](https://user-images.githubusercontent.com/87136903/146182973-a31668d3-2fe8-41d8-b328-56db8782a3d8.png)

Here is how it looks like on my data : I don't think it makes sense to align on the max isi, especially for the 'align on both taps' column.

Thanks in advance!
Clara

**Comment by ahwillia** [comment]
Yes its possible, but you'll have to work out the math. See the docstring for [`manual_fit`](https://github.com/ahwillia/affinewarp/blob/master/affinewarp/piecewisewarp.py#L378). It may be as easy as replacing `tmax` in the example notebook with whatever median or mean that you'd like to use as the anchor.
